### PR TITLE
Retry for queries related to schema modifications in CI

### PR DESCRIPTION
**Pull request by yixinglu**
as title

### CODE REVIEW

**Review by Aiee — DISMISSED** [review]

File: tests/tck/features/schema/Schema.feature [hunk 1]
```diff
@@ -399,9 +402,8 @@ Feature: Insert string vid of vertex and edge
       CREATE TAG animal(name string, kind string);
       """
     Then the execution should be successful
-    Given wait 3 seconds
     # check result
-    When executing query:
+    When try to execute query:
```
Comment (laura-ding) [inline tests/tck/features/schema/Schema.feature:406]: here not need `try`

File: tests/tck/features/ttl/TTL.feature [hunk 2]
```diff
@@ -12,8 +12,7 @@ Feature: TTLTest
       """
       CREATE TAG person(name string, email string, age int, gender string, row_timestamp timestamp);
       """
-    And wait 6 seconds
-    When executing query:
+    When try to execute query:
```
Comment (laura-ding) [inline tests/tck/features/ttl/TTL.feature:15]: ditto

File: tests/tck/features/ttl/TTL.feature [hunk 3]
```diff
@@ -196,9 +195,8 @@ Feature: TTLTest
       """
       CREATE EDGE work1(name string, email string, age int, gender string, row_timestamp timestamp) ttl_duration = 100, ttl_col = "row_timestamp";
       """
-    And wait 6 seconds
     Then the execution should be successful
-    When executing query:
+    When try to execute query:
```
Comment (laura-ding) [inline tests/tck/features/ttl/TTL.feature:199]: ditto

File: tests/tck/features/ttl/TTL.feature [hunk 4]
```diff
@@ -98,8 +97,8 @@ Feature: TTLTest
       """
       ALTER TAG woman Drop (name) ttl_duration = 200;
       """
-    And wait 6 seconds
     Then the execution should be successful
+    And wait 6 seconds
```
Comment (laura-ding) [inline tests/tck/features/ttl/TTL.feature:101]: delete it

File: tests/tck/features/ttl/TTL.feature [hunk 5]
```diff
@@ -121,8 +120,8 @@ Feature: TTLTest
       """
       ALTER TAG woman Drop (row_timestamp);
       """
-    And wait 6 seconds
     Then the execution should be successful
+    And wait 6 seconds
```
Comment (laura-ding) [inline tests/tck/features/ttl/TTL.feature:124]: ditto

File: tests/tck/features/ttl/TTL.feature [hunk 6]
```diff
@@ -134,8 +133,8 @@ Feature: TTLTest
       """
       ALTER TAG woman ttl_duration = 100, ttl_col = "age";
       """
-    And wait 6 seconds
     Then the execution should be successful
+    And wait 6 seconds
```
Comment (laura-ding) [inline tests/tck/features/ttl/TTL.feature:137]: ditto

File: tests/tck/features/schema/Schema.feature (hunk 1 above)
Comment (yixinglu) [reply tests/tck/features/schema/Schema.feature:406]: Will the `DESCRIBE` command ask the metad the schema information ?

Comment (laura-ding) [reply tests/tck/features/schema/Schema.feature:406]: Yes

Comment (yixinglu) [reply tests/tck/features/schema/Schema.feature]: fixed

**Review by Aiee — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**